sources/assets/Stride.Core.Assets/PackageSession.Dependencies.cs

```diff
@@ -396,26 +396,6 @@ public static void UpdateDependencies(SolutionProject project, bool directDepend
             }
         }
 
-        private static RemoteWalkContext CreateRemoteWalkContext(RestoreRequest request, RestoreCollectorLogger logger)
-        {
-            var context = new RemoteWalkContext(
-                request.CacheContext,
-                request.PackageNameSpaces,
-                logger);
-
-            foreach (var provider in request.DependencyProviders.LocalProviders)
-            {
-                context.LocalLibraryProviders.Add(provider);
-            }
-
-            foreach (var provider in request.DependencyProviders.RemoteProviders)
-            {
-                context.RemoteLibraryProviders.Add(provider);
-            }
-
-            return context;
-        }
-
```

Comment:
One of the props wasn't part of the API anymore and this function isn't used anywhere.